jetbrains-ultimate/src/software/aws/toolkits/jetbrains/datagrip/auth/SecretsManagerAuthWidget.kt

```diff
@@ -26,6 +27,7 @@ class SecretsManagerAuthWidget : AwsAuthWidget(userFieldEnabled = false) {
 
     override val rowCount = 5
     override fun getRegionFromUrl(url: String?): String? = RdsResources.extractRegionFromUrl(url) ?: RedshiftUtils.extractRegionFromUrl(url)
+    override val serviceId: String = SecretsManagerClient.SERVICE_NAME
```

Comment:
For IAM we say RDS, but for this we say SecretsManager?

Reply:
We can't tell which service it's for since it supports Redshift and RDS